suite-native/module-device-onboarding/src/components/WalletCreationAccordionHint.tsx

```diff
@@ -1,48 +1,62 @@
-import { ReactNode } from 'react';
+import { ReactNode, useEffect, useState } from 'react';
 import { FadeInDown } from 'react-native-reanimated';
 
 import { AccordionList, AnimatedCard, Text } from '@suite-native/atoms';
 import { IconName } from '@suite-native/icons';
-import { Translation, TxKeyPath } from '@suite-native/intl';
+import { Translation } from '@suite-native/intl';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-const AccordionContentText = ({ translationKey }: { translationKey: TxKeyPath }) => (
-    <Text variant="hint">
-        <Translation id={translationKey} />
-    </Text>
-);
-
 const cardStyle = prepareNativeStyle(utils => ({
     paddingBottom: utils.spacings.sp8,
 }));
 
+const SHOW_ACCORDION_DELAY = 5000;
+
 const accordionItems = [
     {
         title: <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion1.title" />,
         content: (
-            <AccordionContentText translationKey="moduleDeviceOnboarding.walletCreationScreen.accordion1.content" />
+            <Text variant="hint">
+                <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion1.content" />
+            </Text>
         ),
         iconName: 'newspaper',
     },
     {
         title: <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion2.title" />,
         content: (
-            <AccordionContentText translationKey="moduleDeviceOnboarding.walletCreationScreen.accordion2.content" />
+            <Text variant="hint">
+                <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion2.content" />
+            </Text>
         ),
         iconName: 'pencilSimple',
     },
     {
         title: <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion3.title" />,
         content: (
-            <AccordionContentText translationKey="moduleDeviceOnboarding.walletCreationScreen.accordion3.content" />
+            <Text variant="hint">
+                <Translation id="moduleDeviceOnboarding.walletCreationScreen.accordion3.content" />
+            </Text>
         ),
         iconName: 'magnifyingGlass',
     },
 ] as const satisfies { title: ReactNode; content: ReactNode; iconName: IconName }[];
 
-export const WalletCreationHint = () => {
+export const WalletCreationAccordionHint = () => {
     const { applyStyle } = useNativeStyles();
 
+    const [isDisplayed, setIsDisplayed] = useState(false);
+
+    useEffect(() => {
+        const timeout = setTimeout(() => {
+            setIsDisplayed(true);
+        }, SHOW_ACCORDION_DELAY);
+
+        return () => clearTimeout(timeout);
+    }, []);
+
+    if (!isDisplayed) return null;
```

Comment:
what about applying the delay effect directly on the `FadeInDown` entering animation? instead of using `useState` hook?